app/controllers/tagging_spreadsheets_controller.rb

```diff
@@ -15,7 +15,7 @@ def create
     if tagging_spreadsheet.valid?
       tagging_spreadsheet.save!
       InitialTaggingImport.perform_async(tagging_spreadsheet.id)
-      redirect_to tagging_spreadsheets_path
+      redirect_to tagging_spreadsheet, success: "Spreadsheet uploaded. It will be imported within a few minutes."
```

Comment:
I think that's one more file to maintain and do search/replace. I don't think we are getting a big benefit from it since we are not translating content.